.travis.yml

```diff
@@ -44,6 +44,8 @@ install:
   -o gae-download.zip
 - unzip -qq gae-download.zip -d $TOOLS_DIR/google_appengine_1.9.19/
 - rm gae-download.zip
+- bash scripts/install_third_party.sh
+- python scripts/pre_commit_linter.py --path=. --silent
```

Comment:
I think it'd be better to run this as a separate check instead of the default installation process, just like we run the backend tests and the frontend tests as a separate check. What do you think?